internal/daemon/worker/auth_rotation_test.go

```diff
@@ -126,6 +131,11 @@ func TestRotationTicking(t *testing.T) {
 			nodeenrollment.WithStorageWrapper(w.Config().WorkerAuthStorageKms),
 		)
 		require.NoError(err)
+		if bytes.Equal(currKey, currNodeCreds.CertificatePublicKeyPkix) {
+			// Load due to parallel tests may mean that we didn't hit rotation
+			// yet, loop back to evaluate again
+			continue
+		}
```

Comment:
Is this a rogue test fix for this flake in the middle of this PR? Mind putting it in a separate PR?